CMakeLists.txt

```diff
@@ -587,6 +588,91 @@ if (BUILD_TESTING)
             set_property(TEST ${test_target} PROPERTY TIMEOUT 7200)
         endforeach()
     endif()
+
+    if(S2N_FUZZ_TEST)
```

Comment:
feature request: `message(STATUS "Fuzz build enabled")`

Reply:
added this in the new commit!